bookmarks.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header class="app-title">
+      <h1>Quiz App</h1>
+    </header>
+    <main>
+      <section class="quiz-card">
+        <img
+          src="img/bookmark-solid.svg"
+          alt="Bookmark Icon"
+          class="quiz-bookmark"
+        />
+        <h2 class="quiz-question">What is Lorem?</h2>
+        <div class="show-answer">
+          <button class="show-answer-button" type="button">Show Answer</button>
+        </div>
+        <p class="quiz-answer">
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem
+          dignissimos fugiat ipsa dolorum voluptatibus nihil suscipit minima!
+          Minima quisquam repellendus exercitationem ipsam, quos enim tempora
+          fugit magni veniam impedit neque?
+        </p>
+        <button class="quiz-tag" type="button">#Lorem</button>
+        <button class="quiz-tag" type="button">#placeholder</button>
+        <button class="quiz-tag" type="button">#syntax</button>
+      </section>
```

Comment:
```suggestion
      <section class="quiz-card">
      <img src="img/bookmark-regular.svg" alt="Bookmark Icon" class="quiz-card__icon" />
      <p class="quiz-card__question">What is Lorem?</p>
      <div class="quiz-card__answer-container">
        <button class="quiz-card__answer-button" type="button">Show Answer</button>
      </div>
      <p class="quiz-card__answer">
        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem
        dignissimos fugiat ipsa dolorum voluptatibus nihil suscipit minima!
        Minima quisquam repellendus exercitationem ipsam, quos enim tempora
        fugit magni veniam impedit neque?
      </p>
      <button class="quiz-card__tag" type="button">#Lorem</button>
      <button class="quiz-card__tag" type="button">#placeholder</button>
      <button class="quiz-card__tag" type="button">#syntax</button>
    </section>
```

Here I give you a example, how you could structure your css with the BEM methodology. And I would change the h2 to an p tag, because you put a question into it and doesn't use it as a headline.